app/controllers/jobs_controller.rb

```diff
@@ -21,24 +21,36 @@ def create
   end
 
   def show
+    @edit_comment = params[:edit_comment] unless params[:edit_comment].nil?
+    @company = Company.find(params[:company_id])
     @job = Job.find(params[:id])
+    @comment = Comment.new
+    @tags = Tag.all - @job.tags
```

Comment:
this seems like a lot of logic for the controller - could we maybe create a helper method for this? or even a method in `Job` that grabs available tags?